packages/testgap/src/06-write.ts

```
@@ -37,15 +37,19 @@ function uuid() {
 async function main() {
   const plans = JSON.parse(
-    await fs.readFile(path.resolve(args["--plans"]), "utf-8"),
+    await fs.readFile(
+      path.resolve(args["--plans"] ?? ".cache/testgap/plans.json"),
+      "utf-8",
+    ),
   ) as PlanFile;
-  await fs.mkdir(path.resolve(args["--out"]), { recursive: true });
+  const outDir = path.resolve(args["--out"] ?? "docs/agile/tasks/test-gaps");
+  await fs.mkdir(outDir, { recursive: true });
   const index: string[] = ["# Test gap tasks", ""];
```

ESLint

[error] Variable should have an immutability of at least "ReadonlyDeep" (actual: "Mutable").

(functional/prefer-immutable-types)